src/test/java/edu/ucsb/cs156/rec/web/SwaggerWebIT.java

```diff
@@ -60,31 +60,4 @@ public void swagger_page_can_be_loaded() throws Exception {
                 .isVisible();
     }
 
-    /**
```

Comment:
Can we restore this, but instead of using Restaurant, use one of the API endpoints that is still there in Swagger?

For example, the User, or System Info endpoints.